### DISCUSSION TITLE
Integration with Home Assistant

### DISCUSSION THREAD

**Discussion by elecnix** [focustimerhq/FocusTimer#675] [Show and tell]
Hello!

I wrote a script to control my lights with [Home Assistant](https://www.home-assistant.io/), so I know I'm supposed to be focusing, or that I'm in a break.

Check it out:

https://github.com/elecnix/gnome-pomodoro-scenes

@kamilprusko Would you be OK to add this to the Wiki, under [Integrations](https://github.com/gnome-pomodoro/gnome-pomodoro/wiki/Integration)?

**Comment by kamilprusko** [comment]
Awesome! Thanks @elecnix :)

Added to wiki.
